python/tvm/contrib/hexagon/transform.py

```diff
@@ -311,3 +318,73 @@ def remove_empty_pad(mod):
     """Remove the empty pad operator."""
     mod["main"] = rewrite(remove_empty_pad_callback(), mod["main"])
     return mod
+
+
+class simplify_qnn_concat_in_func(DFPatternCallback):
+
+    """
+    Propagate qnn.concat's quantization params to its inputs,
+    and try to avoid redundant requantization while doing so.
+    """
```

Comment:
Minor comment. If possible we could add a snippet of the relay IR before and after this pass in the comments similar to the comments mentioned in the functions above (like `remove_empty_pad_callback` or `qdistilbert_rewrite`).